src/storage/benches/bench_compactor.rs

```diff
@@ -220,15 +220,15 @@ fn bench_merge_iterator_compactor(c: &mut Criterion) {
     let sstable_store = mock_sstable_store();
     let test_key_size = 256 * 1024;
     let info1 = runtime
-        .block_on(async { build_table(sstable_store.clone(), 1, 0..test_key_size, 1).await });
+        .block_on(async { build_table(sstable_store.clone(), 1, 0..test_key_size/2, 1).await });
```

Comment:
Why divide key size by 2?

Reply:
Because it would cause panic because of duplicated key